File: hardware/Test Relay/Relay/Relay.cpp

#include <Relay.h>

Relay::Relay() {
    port=0;
    number=0;
    status=0;
    time=0;
    time_start="";
    time_finish="";
    automatic= false;    
}

int Relay::getPort(){
    return port;
}

void Relay::setPort(int newPort){
    port=newPort;
}

int Relay::getNumber(){
    return number;
}

void Relay::setNumber(int newNumber){
     number=newNumber;
}

int Relay::getStatus(){
    return status;
}

void Relay::setStatus(int newStatus){
     status=newStatus;
}

String Relay::getStartTime (){
        return time_start;
}
void Relay::setStartTime (String time){
     time_start=time;
}
         
String Relay::getEndTime (){
     return time_finish;
     }
     
void Relay::setEndTime (String time){
     time_finish=time;
}

bool Relay::getAutomatic(){
     return automatic;
}

void Relay::setAutomatic(bool newAutomatic){
     automatic=newAutomatic;
}    

File: app/mobile/app/src/main/java/com/test/firsttestfirebase/Relay.java
package com.test.firsttestfirebase;

public class Relay {
    int relayNumber;
    int status;
    int time;
    String time_start;
    String time_finish;
    boolean automatic;


    public Relay() {

        relayNumber = 0;
        status = 0;
        time = 0;
        time_start = "";
        time_finish = "";
        automatic = false;

    }

    int getRelayNumber() {
        return relayNumber;
    }

    void setRelayNumber(int newNumber) {
        relayNumber = newNumber;
    }

    int getStatus() {
        return status;
    }

    void setStatus(int Status) {
        this.status = status;
    }

    String getStartTime() {
        return time_start;
    }

    void setStartTime(String time) {
        time_start = time;
    }

    String getEndTime() {
        return time_finish;
    }

    void setEndTime(String time) {
        time_finish = time;
    }

    boolean getAutomatic() {
        return this.automatic;
    }

    void setAutomatic(boolean automatic) {
        this.automatic = automatic;

    }
}
